core/src/test/scala/org/allenai/common/cache/QueryCacheSpec.scala

```diff
@@ -1,17 +1,27 @@
 package org.allenai.common.cache
 
+import java.io.File
 import org.allenai.common.{ GitVersion, Version, cache }
 import org.allenai.common.testkit.UnitSpec
 import org.scalatest.BeforeAndAfterAll
-import sys.process._
+import spray.json.DefaultJsonProtocol
 import spray.json.DefaultJsonProtocol._
+import spray.json._
+import sys.process._
+
+case class Foo(stringVar: String, intVar: Int)
+
+object FooJsonProtocol extends DefaultJsonProtocol {
+  implicit val fooFormat = jsonFormat2(Foo)
+}
 
 class QueryCaches(redisHostname: String, redisPort: Int) {
   val stringQueryCache = new JsonQueryCache[String](redisHostname, redisPort, "test")
   val intQueryCache = new JsonQueryCache[Int](redisHostname, redisPort, "test")
   val seqStringQueryCache = new JsonQueryCache[Seq[String]](redisHostname, redisPort, "test")
   // It's an object I can test
-  val versionQueryCache = new JsonQueryCache[Version](redisHostname, redisPort, "test")
+  import FooJsonProtocol._
+  val fooQueryCache = new JsonQueryCache[Foo](redisHostname, redisPort, "test")
```

Comment:
Does this talk over the network? If so, it should be under `it` (and maybe not run automatically).